whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceTestIntegration.java

```diff
@@ -5750,6 +5767,59 @@ public void update_route6_in_region_no_redirect() {
 
     }
 
+    @Test
+    public void unsubscribed_notify_user_gets_warn_when_updating() {
+        final String unsubscribedEmail = "test@ripe.net";
```

Comment:
Can you either update existing or create new one where atleast one  notify email where it is not  unsubscribed or undelieverd and thats why no warning for that email address

Reply:
we have a bunch of test that are already doing that, right? updates that are checking the warnings already in this warning doesn't exist among them